LEAF_Request_Portal/form.php

```diff
@@ -3063,15 +3068,31 @@ public function query($inQuery)
 
         if ($joinRecords_Step_Fulfillment)
         {
-            $res2 = $this->db->prepared_query('SELECT * FROM records_step_fulfillment
-    											LEFT JOIN workflow_steps USING (stepID)
-    											WHERE recordID IN (' . $recordIDs . ')', array());
+            $strSQL = 'SELECT * FROM records_step_fulfillment LEFT JOIN workflow_steps USING (stepID) '.
```

Comment:
Please change this so it's not "select *".

Reply:
is this an update query??